lib/backpex/item_actions/item_action.ex

```diff
@@ -22,6 +22,8 @@ defmodule Backpex.ItemAction do
   @callback fields() :: list()
 
   @doc """
+  TODO: rename to `init_schema`
```

Comment:
I want to rename `init_change`. You might think that this function passes the result as the change / params / attrs into the changeset, but it is used as the source changeset data. WDYT, is `init_schema` a good name? @pehbehbeh

Reply:
See https://github.com/naymspace/backpex/pull/672